src/OrchardCore/OrchardCore.Localization.Core/PortableObject/PortableObjectStringLocalizerFactory.cs

```diff
@@ -60,5 +65,14 @@ public IStringLocalizer Create(string baseName, string location)
 
             return new PortableObjectStringLocalizer(relativeName, _localizationManager, _fallBackToParentCulture, _logger);
         }
+
+        // The context within inner class
+        private void TryFixInnerClassPath(ref string context)
```

Comment:
Can you try to just return the new string? The `ref` should not be necessary in this case (just makes it more copmlex to read). Also missing space after `if`

Reply:
> Can you try to just return the new string?

Ok

> Also missing space after if

Good catch :)